help/en/html/tools/Logix.shtml

```diff
@@ -361,6 +361,10 @@
         <li>if (antecedent state changes) then (do "on change" actions)</li>
       </ul>
 
+      <p> Note that the state of any antecedent is considered to have changed when a Logix is first enabled. This
+      means that any "do on change" actions will be executed when a Logix is first enabled, including those tha
+      t  are enabled when JMRI is started</p>
```

Comment:
Not important, but I noticed that `that` has a new line in it.
```
including those tha
t  are
```

Reply:
Fixed and submitted as PR #10459.  Any idea on how to remove my changes to Logix.shtml from this PR?  I am only using github.com (not the desktop version).

Reply:
Thanks.
I removed the section so these changes are now removed.